domain-ee/ee-ep-merge-app/src/python_src/service/ep_merge_machine.py

```diff
@@ -230,6 +235,16 @@ def on_get_pending_contentions(self, event):
             response_type=get_contentions.Response,
             expected_statuses=expected_statuses,
         )
+        if response and response.status_code in expected_statuses:
+            contentions = response.contentions or []
+            num_pending_ep_new_contentions = sum(1 for c in contentions if c.contention_type_code == 'NEW')
+            num_pending_ep_cfi_contentions = sum(1 for c in contentions if c.contention_type_code == 'INCREASE')
```

Comment:
Booleans convert to 0 and 1, so these can be rephrased like

```python
sum(c.contention_type_code == 'NEW' for c in contentions)
```

which is not a strong convention, but I think it reads fine if you think of `sum` like "count."